NEWS.md

```diff
@@ -2,6 +2,7 @@
 
 ## Deprecations & Breaking Changes
 
+* .lintr configs can now be kept in the directory .github/linters for better compatibility with Super-Linter. Note that this may be a breaking change if you already have a config in both such a directory and in your R project's root since the former will now be discovered first where it was ignored before. Please see vignette("lintr") for details on how configs are discovered. (#1746, @tonyk7440)
```

Comment:
IINM, the only breaking constellation is if `.github/linter/.lintr` is in a subdirectory, right?

i.e.
```
.lintr
tests/.github/linters/.lintr
```

will find a different config when run from `tests/`. Is that correct?

Reply:
You’re absolutely right! Will amend.